src/fsharp/CheckFormatStrings.fs

```diff
@@ -117,13 +115,13 @@ let parseFormatStringInternal (m:range) (g: TcGlobals) (context: FormatStringChe
               let rec digitsPrecision i = 
                 if i >= len then failwithf "%s" <| FSComp.SR.forBadPrecision()
                 match fmt.[i] with
-                | c when isDigit c -> digitsPrecision (i+1)
+                | c when System.Char.IsDigit c -> digitsPrecision (i+1)
```

Comment:
Maybe, open the namespace instead?

Reply:
More often than not we use System.Char.IsDigit in this codebase.  It's not clear to me why.

Reply:
And amusingly @forki's PR does it both ways.